viz_scripts/scaffolding.py

```diff
@@ -495,29 +495,28 @@ def extract_footprint(footprint_dict, footprint_key):
     else:
         return np.nan
 
-def unpack_energy_emissions(expanded_ct):
-    expanded_ct['mode_confirm_kg_CO2'] = expanded_ct['mode_confirm_footprint'].apply(extract_footprint, footprint_key='kg_co2')
-    expanded_ct['mode_confirm_lb_CO2'] = kg_to_lb(expanded_ct['mode_confirm_kg_CO2'])
-    expanded_ct['replaced_mode_kg_CO2'] = expanded_ct['replaced_mode_footprint'].apply(extract_footprint, footprint_key='kg_co2')
-    expanded_ct['replaced_mode_lb_CO2'] = kg_to_lb(expanded_ct['replaced_mode_kg_CO2'])
+def unpack_energy_emissions(expanded_ct, use_imperial):
```

Comment:
nit, future fix: why do we need to pass this in? It looks like, before this, we used to always compute both `kg` and `lb`; why can't we continue doing that?